samples/IdentitySample.Mvc/Migrations/20151110211659_Identity2-3.cs

```diff
@@ -0,0 +1,155 @@
+using System;
+using System.Collections.Generic;
+using Microsoft.Data.Entity.Migrations;
```

Comment:
Ah, this looks like a migration done using EF7. Weren't you going to use EF6 to upgrade the database schema?